shaded/ozone/pom.xml

```diff
@@ -62,15 +63,26 @@
       </activation>
       <dependencies>
         <dependency>
-          <groupId>org.apache.hadoop</groupId>
-          <artifactId>hadoop-ozone-filesystem</artifactId>
+          <groupId>org.apache.ozone</groupId>
+          <artifactId>ozone-filesystem</artifactId>
```

Comment:
`hadoop-ozone-filesystem` has been changed to `ozone-filesystem-hadoop2` and `ozone-filesystem-hadoop3`

Reply:
please reference this PR https://github.com/Alluxio/alluxio/pull/14672